validator_notifier/src/lambda_function.py

```diff
@@ -0,0 +1,114 @@
+import base64
+import json
+import logging
+import os
+
+from mailing import Mailer
+from signing import Signer
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+MAX_MESSAGE_LENGTH = 4096
+
+
+class MessageTooLongError(Exception):
+    pass
+
+
+def run(event, context):
+    try:
+        ensure_authentication(event)
+    except Exception as e:
+        logger.info(f"Authentication failed: {e}")
+        return {"statusCode": 403}
+
+    try:
+        subject, unified_message, decorated_content = validate_input(event)
+    except MessageTooLongError as e:
+        logger.info("Message is too long")
+        return {
+            "statusCode": 400,
+            "body": f"Message is too long. Maximum length is {MAX_MESSAGE_LENGTH}.",
```

Comment:
Nice!
Let's add the max length to the README too

Reply:
Good idea. Done!